zh/configure-a-tidb-cluster.md

```diff
@@ -1,34 +1,288 @@
 ---
-title: Kubernetes 上的 TiDB 集群的资源配置以及容灾配置
-summary: 介绍 Kubernetes 上 TiDB 集群的的资源配置以及容灾配置。
-category: reference
+title: 配置 TiDB 集群
+summary: 了解如何在 Kubernetes 中配置 TiDB 集群。
+category: how-to
```

Comment:
```suggestion
category: how-to
aliases: ['/zh/tidb-in-kubernetes/dev/configure-cluster-using-tidbcluster/','/docs-cn/tidb-in-kubernetes/dev/configure-cluster-using-tidbcluster/]
```
Add these aliases to avoid 404 error when users access the deleted file.